hw/ip/otbn/dv/otbnsim/sim/insn.py

```diff
@@ -714,7 +714,7 @@ def execute(self, model: OTBNModel) -> None:
         a = int(model.state.wreg[self.wrs1])
         b_shifted = ShiftReg(int(model.state.wreg[self.wrs2]), self.shift_type,
                              self.shift_bytes)
-        carry_flag = 1 - model.state.flags[self.flag_group].C
+        carry_flag = -model.state.flags[self.flag_group].C
```

Comment:
I think the naming was already wrong, but surely `0` / `-1` shouldn't be called a carry "flag". Should it be something like "carry in"?

Reply:
I agree, carry_in or something like that would be clearer. Whatever you change it to, please also change the pseudo-code in the OTBN docs.